uint32 id
Position pos
Route route
float64 inclination
float64 bearing
float64 speed
bool landed
